nilearn/datasets/description/fsaverage6.rst

```diff
@@ -0,0 +1,21 @@
+fsaverage6
+
+
+Notes
+-----
+Fsaverage6 standard surface as distributed with Freesurfer (Fischl et al, 1999)
+
+Content
+-------
+    :'pial_left': Gifti file, left hemisphere pial surface mesh
+    :'pial_right': Gifti file, right hemisphere pial surface mesh
+    :'infl_left': Gifti file, left hemisphere inflated pial surface mesh
+    :'infl_right': Gifti file, right hemisphere inflated pial
+                   surface mesh
+    :'sulc_left': Gifti file, left hemisphere sulcal depth data
+    :'sulc_right': Gifti file, right hemisphere sulcal depth data
```

Comment:
and also white matter?

Reply:
Yes, someone mentioned it in #2313. I might integrate the white surface in this PR, or not, we'll see :)

Reply:
white matter for fsaverage (aka fsaverage7) and fsaverage5 are aready downloaded by the existing fetchers, so you only need to worry about fsaverage6! #2313 had remained open by mistake AFAICT

Reply:
However, maybe we could add the curvature to `fsaverage` and `fsaverage5`, what do you think?